src/components/FlawCreateForm.vue

```diff
@@ -93,10 +88,10 @@ const incidentTypes = Object.values(ZodFlawSchema.shape.major_incident_state.unw
           <div class="col-6 text-end">
             <div class="row">
               <div class="col-6">
-                <div class="input-group mb-2" :class="{major: modelValue.is_major_incident, 'bg-danger': modelValue.is_major_incident}">
-                  <span class="input-group-text">Major Incident:</span>
-                  <select class="form-control" v-model="modelValue.type">
-                    <option v-for="incidentType in incidentTypes" :value="incidentType">{{incidentType}}</option>
+                <div class="input-group mb-2" :class="{major: modelValue.major_incident_state}">
```

Comment:
I expect that this doesn't work due to the change from boolean -> string; update the evaluation to become a boolean when the incident state is major again